person_capture/gui_app.py

```
@@ -4638,7 +4710,8 @@ def save_hit(c, idx):
                             # skip this hit, keep processing
                             pass
                     if hdr_out_path:
-                        self._save_hdr_crop_p010(idx, (cx1, cy1, cx2, cy2), hdr_out_path)
+                        hdr_crop_xyxy = self._even_hdr_crop_xyxy(source_crop_xyxy, source_size)
+                        self._save_hdr_crop_p010(idx, frame_pts_sec, hdr_crop_xyxy, hdr_out_path)
```

Ruff (0.15.11)

[warning] Function definition does not bind loop variable `frame_pts_sec`

(B023)